finality-aleph/src/sync/service.rs

```diff
@@ -585,20 +586,24 @@ where
         match event {
             BlockImported(header) => {
                 trace!(target: LOG_TARGET, "Handling a new imported block.");
+                let mut own_block = false;
+                let id = header.id();
                 self.metrics.report_event(Event::HandleBlockImported);
                 match self.handler.block_imported(header) {
-                    Ok(Some(broadcast)) => {
-                        if let Err(e) = self
-                            .network
-                            .broadcast(NetworkData::RequestResponse(broadcast))
-                        {
-                            warn!(
-                                target: LOG_TARGET,
-                                "Error broadcasting newly created block: {}.", e
-                            )
-                        };
+                    Ok(block_to_broadcast) => {
+                        if let Some(broadcast) = block_to_broadcast {
+                            own_block = true;
```

Comment:
Why the change from a match arm to an internal if? Not much wrong with that, question mostly out of curiosity.